### DISCUSSION TITLE
How can I remove the ComboBox's underline?

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [AvaloniaCommunity/Material.Avalonia#412] [Q&A] [answered]
How can I remove this line at the bottom of the ComboBox? 

![image](https://github.com/user-attachments/assets/63bd8418-1573-4708-a5c5-c672c92a31de)

I don't have much experience in Styles or templates... I see that line should be there due to the presence of "MaterialUnderline" [here](https://github.com/AvaloniaCommunity/Material.Avalonia/blob/9e7758bf9ac7599352b206b7c07d4334b8a45075/Material.Styles/Resources/Themes/ComboBox.axaml#L92). What is the right approach to remove it?

**Comment by SKProCH** [comment, marked as answer]
Hello. Probably the easy way is to copy the full ComboBox ControlTheme to your app, give it an unique key, remove the underline and use it. 

The other way is probably hiding the underline via style with selector like this: `ComboBox /template/ namespace|MaterialUnderline#PART_Underline`, and setter to Opacity or IsVisible.

**Reply by alessandrofrancesconi** [reply]
It worked via selector! Thanks
